.github/workflows/publish-release.yml

```diff
@@ -45,19 +45,9 @@ jobs:
         run: |
           ./gradlew release -Prelease.useAutomaticVersion=true
           ./gradlew -Pversion=${VERSION} publish -x test -PpublishToCentral=true
-      - name: Create Github release from the release tag
-        run: |
-          curl --request POST 'https://api.github.com/repos/ballerina-platform/module-ballerina-sql/releases' \
-          --header 'Accept: application/vnd.github.v3+json' \
-          --header 'Authorization: Bearer ${{ secrets.BALLERINA_BOT_TOKEN }}' \
-          --header 'Content-Type: application/json' \
-          --data-raw '{
-              "tag_name": "v'"$VERSION"'",
-              "name": "module-ballerina-sql-v'"$VERSION"'"
-          }'
-      - name: Post release PR
+      - name: Post release tasks
```

Comment:
This is not exactly `Post Release Tasks` it seems? May be `Github Release`?

Reply:
Github Release and Release sync PR?

Reply:
+1

Reply:
Fixed in efafc55